--- vendor/studio-42/elfinder/php/elFinder.class.php.orig	2026-04-25 01:13:55
+++ vendor/studio-42/elfinder/php/elFinder.class.php	2025-08-28 11:51:22
@@ -4853,11 +4853,11 @@
         static $defLimit = null;
         if (!self::aborted()) {
             if (is_null($defLimit)) {
-                $defLimit = (int)ini_get('max_execution_time');
+                $defLimit = ini_get('max_execution_time');
             }
             if ($defLimit != 0) {
                 $time = is_null($time) ? $defLimit : max($defLimit, $time);
-                set_time_limit((int)($time));
+                set_time_limit($time);
             }
         } else {
             throw new elFinderAbortException();
