--- vendor/studio-42/elfinder/php/elFinder.class.php.orig
+++ vendor/studio-42/elfinder/php/elFinder.class.php
@@ -4856,7 +4856,7 @@
                 $defLimit = (int)ini_get('max_execution_time');
             }
             if ($defLimit != 0) {
-                $time = is_null($time) ? $defLimit : max($defLimit, $time);
+                $time = is_null($time) ? (int)$defLimit : max((int)$defLimit, (int)$time);
                 set_time_limit($time);
             }
         } else {
